include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -960,7 +960,13 @@ class CommonOperatorLoader : public ProtobufLoader {
       RETURN_IF_ERR(loadIdentity(op, dict));
       return true;
     }
-    if (typeName == "Identity") {
+    if (typeName == "EnforceFinite") {
+      // Load as Identity operation, because in Caffe2 this op does not change
+      // any data. It only raises error if some tensor element is Inf or NaN.
+      RETURN_IF_ERR(loadIdentity(op, dict));
+      return true;
+    }
+    if (typeName == "Identity" || "Alias") {
```

Comment:
Does this really work? You don't need `typeName == "Identity" || typeName ==  "Alias"`